### ISSUE TITLE
[BUG] Rubocop Fixes

### ISSUE THREAD

**Issue by rhuanbarreto** [OpenAPITools/openapi-generator#5417]
##### Description

Rubocop is giving the following error:

```
Error: The `Layout/IndentFirstArgument` cop has been renamed to `Layout/FirstArgumentIndentation`.
(obsolete configuration found in clients/ruby/.rubocop.yml, please update it)
The `Layout/TrailingBlankLines` cop has been renamed to `Layout/TrailingEmptyLines`.
(obsolete configuration found in clients/ruby/.rubocop.yml, please update it)
The `Style/UnneededPercentQ` cop has been renamed to `Style/RedundantPercentQ`.
(obsolete configuration found in clients/ruby/.rubocop.yml, please update it)
The `Style/BracesAroundHashParameters` cop has been removed.
(obsolete configuration found in clients/ruby/.rubocop.yml, please update it)
obsolete `EnforcedStyle: rails` (for Layout/IndentationConsistency) found in clients/ruby/.rubocop.yml
`EnforcedStyle: rails` has been renamed to `EnforcedStyle: indented_internal_methods`
```
##### openapi-generator version

4.2.3

**Comment by ackintosh** [comment]
@rhuanbarreto Thanks for this issue. 😃 Would you please provide your OpenAPI declaration file content which produces the issue?

**Comment by rhuanbarreto** [comment]
You can use the petstore API definition in http://petstore.swagger.io:8080/api/v3/openapi.json.

Command to generate:
```
#!/bin/sh

export RUBY_POST_PROCESS_FILE="/usr/local/bin/rubocop -a"
openapi-generator generate -g ruby -o ruby -i ./openapi.json
```

If you fix the errors by hand, then you have the following error:

```
test $ rubocop
Inspecting 33 files
.....................C...........

Offenses:

ruby/spec/api_client_spec.rb:159:13: C: Layout/SpaceBeforeBlockBraces: Space missing to the left of {.
      expect{api_client.build_collection_param(param, :INVALID)}.to raise_error(RuntimeError, 'unknown collection format: :INVALID')
            ^
ruby/spec/api_client_spec.rb:159:14: C: Layout/SpaceInsideBlockBraces: Space missing inside {.
      expect{api_client.build_collection_param(param, :INVALID)}.to raise_error(RuntimeError, 'unknown collection format: :INVALID')
             ^
ruby/spec/api_client_spec.rb:159:64: C: Layout/SpaceInsideBlockBraces: Space missing inside }.
      expect{api_client.build_collection_param(param, :INVALID)}.to raise_error(RuntimeError, 'unknown collection format: :INVALID')
```

**Comment by rhuanbarreto** [comment]
Rubocop version: 0.80.0

**Comment by autopp** [comment]
@rhuanbarreto Thanks for reporting. It should have been resolved with #5474.
